### PR TITLE
Added active storage to the project

### LINKED ISSUE #10: Image upload
Install active storage

- [ ] user should be able to upload an profile image

### PR DESCRIPTION
**Pull request by LDVera**
Active storage was added to the project
into users_controller was generated new methods (edit, update, show)
the user now can upload a profile photo
the avatar param was permited into the user_params
was added relation has_one_attached into the user model
partial_form for user into views/users was modificated to user a form_with
new _edit partial was creted into views/users/ to get data of the user 
was created 2 new files into views/users (edit/show.html.erb)
rails active_storage:install generated a new migration

Co-authored-by: jnegrete15 <jnegret15@ucol.mx>

### CODE REVIEW

File: app/controllers/main_controller.rb [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
-  def index; end
+  def index
+    # @user = User.find(params[:id])
+  end
+
+  def edit
+    
+  end
+  
 end
```
Comment (OchoaHub) [inline app/controllers/main_controller.rb:12]: ¿?

**Review by OchoaHub — APPROVED** [review]
Amazing code! 👍 💯

File: app/controllers/main_controller.rb (hunk 1 above)
Comment (LDVera) [reply app/controllers/main_controller.rb:12]: forget undo that unnecessary changes, sorry!

**Review by LPMendoza — APPROVED** [review]
God job man, just delete the unnecessary changes

**Review by MiguelOcegueraM — APPROVED** [review]

**Pull request merged**